docs/_docs/0.8.0/1_1_spark_quick_start_guide.md

```diff
@@ -0,0 +1,530 @@
+---
+version: 0.8.0
+title: "Quick-Start Guide"
+permalink: /docs/0.8.0-spark_quick-start-guide.html
+toc: true
+last_modified_at: 2019-12-30T15:59:57-04:00
+---
+
+This guide provides a quick peek at Hudi's capabilities using spark-shell. Using Spark datasources, we will walk through 
+code snippets that allows you to insert and update a Hudi table of default table type: 
+[Copy on Write](/docs/0.8.0-concepts.html#copy-on-write-table). 
+After each write operation we will also show how to read the data both snapshot and incrementally.
+# Scala example
+
+## Setup
+
+Hudi works with Spark-2.x & Spark 3.x versions. You can follow instructions [here](https://spark.apache.org/downloads.html) for setting up spark. 
+From the extracted directory run spark-shell with Hudi as:
+
+```scala
+// spark-shell
+spark-shell \
```

Comment:
call out that this is spark2 bundle. if someone wants to try out w/ spark3, its hudi-spark3-bundle_2.12:0.7.0.
oh, btw, you need to fix the version no here to 0.8.0

Reply:
good catch